src/main/ipc.js

```diff
@@ -61,6 +61,14 @@ function init (cwd, state) {
     windows.main.send('DC_EVENT_IMEX_PROGRESS', progress)
   })
 
+  dc.on('DC_EVENT_ERROR', (error) => {
+    windows.main.send('error', error.toString())
+  })
+
+  dc.on('DC_EVENT_NETWORK_ERROR', (error) => {
+    windows.main.send('error', error.toString())
+  })
```

Comment:
We could merge these ones if the delta chat controller emitted the same event.

Reply:
I think it's ok to keep them different for now

Reply:
It's fine :)

Reply:
The `.toString()` shouldn't be needed though.